lib/floe/workflow/context.rb

```diff
@@ -45,6 +45,10 @@ def output
         state["Output"]
       end
 
+      def output=(val)
+        state["Output"] = val
+      end
+
```

Comment:
I admit that code climate complaining about adding too many methods to `Context` shaped my design. So I had a single method that sets the `output`, `next_state`, `error`, and `cause`.

Nice thing about these helper methods is we can move `output` out of `"State"` when we want to remove some of these non-spec values from the hash.